Gemfile

```diff
@@ -0,0 +1,47 @@
+source 'https://rubygems.org'
+
+gem 'rails-erd'
+gem 'ruby-graphviz'
```

Comment:
Yeah - I love rails-erb. One rake command (`rake erd`) and boom - ERD diagram done as PDF in root folder.